modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -112,27 +110,65 @@ function perflab_aea_enqueued_js_assets_test() {
 			'color' => 'blue',
 		),
 		'description' => sprintf(
-		/* translators: 1: Number of enqueued scripts. 2: "script" word. 3.Scripts size. */
-			'<p>' . esc_html__( 'The amount of %1$s enqueued %2$s (size: %3$s) is acceptable.', 'performance-lab' ) . '</p>',
-			$enqueued_scripts,
-			_n( 'script', 'scripts', $enqueued_scripts, 'performance-lab' ),
-			size_format( perflab_aea_get_total_size_bytes_enqueued_scripts() )
+			'<p>%s</p>',
+			esc_html(
+				sprintf(
+				/* translators: 1: Number of enqueued styles. 2.Styles size. */
```

Comment:
Nit-pick, but can you indent one tab more this so that it is aligned with the below `_n` call that it applies to?